docs/src/locators.md

```diff
@@ -186,27 +186,23 @@ The [`method: Page.getByRole`] locator reflects how users and assistive technolo
 
 For example, consider the following DOM structure.
 
-```html
+```html card
 <h3>Sign up</h3>
-<form>
-  <input type="checkbox" id="subscribe" />
-  <label for="subscribe">Subscribe</label>
-  <br>
-  <button>Submit</button>
-</form>
+<label>
+<input type="checkbox" /> Subscribe
+</label>
+<br/>
+<button>Submit</button>
 ```
 
 You can locate each element by it's implicit role:
 
 ```js
-await expect(page.getByRole('heading', { name: 'Sign up' }))
-    .toBeVisible()
+await expect(page.getByRole('heading', { name: 'Sign up' })).toBeVisible()
```

Comment:
I added the chaining .toBeVisible() to next line for all JS examples as so much easier to read and follow along.

Reply:
You did not do that for other languages or in any other places though?

Reply:
I'll reformat all the examples for col:80